checks/pre-commit

```diff
@@ -0,0 +1,7 @@
+set -e
+echo "precommit running. checking build located at ${subdir}."
+cd ${subdir}
+../checks/run-coverage.sh
```

Comment:
I think I was looking at at run-coverage.sh and thought it would make sense to check if it fails and provide an error message if that happened. I also missed the "set -e". It's a simple script so I guess it doesn't matter.